src/app/pages/ArticlePage/experimentTopStories/helpers.tsx

```diff
@@ -108,9 +127,101 @@ export const ExperimentTopStories = ({
     <div
       css={styles.experimentTopStoriesSection}
       data-testid="experiment-top-stories"
-      data-vars-top-stories-position="experiment"
+      data-experiment="position:articleBody"
```

Comment:
**Suggestion (non-blocking)** If you are only using this to track the position of the top stories then you could just do `data-experiment-position="articleBody"` etc